common/src/main/java/com/wynntils/core/notifications/NotificationManager.java

```diff
@@ -29,6 +43,19 @@ public static MessageContainer queueMessage(TextRenderTask message) {
 
         WynntilsMod.info("Message Queued: " + message);
         MessageContainer msgContainer = new MessageContainer(message);
+        String messageText = message.getText();
+        Integer messageTextHash = messageText.hashCode();
+
+        for (Pair<String, MessageContainer> cachedMessagePair : cachedMessageSet) {
+            Integer iteratedMessageHash = cachedMessagePair.a().hashCode();
+            if (messageTextHash.equals(iteratedMessageHash)) {
```

Comment:
Why do we compare the hashes? Why not compare the message content string itself?

Reply:
I had originally had it work on the strings themselves but was under the impression that a hashed integer would be simpler to check again. I thought your recommended something along the line of "hashcode equals" in Discord the other day, is this jot what you meant?

Reply:
> I had originally had it work on the strings themselves but was under the impression that a hashed integer would be simpler to check again. I thought your recommended something along the line of "hashcode equals" in Discord the other day, is this jot what you meant?

No, I did not mean this. I don't see why you couldn't just .equals 2 strings.